exotica_examples/resources/configs/example_ik_eff_box_constraint.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" ?>
+<IKSolverDemoConfig>
+
+  <!--Example uses SciPy solver.-->
+
+  <EndPoseProblem Name="My Problem">
+
+    <PlanningScene>
+      <Scene>
+        <JointGroup>arm</JointGroup>
+        <URDF>{exotica_examples}/resources/robots/lwr_simplified.urdf</URDF>
+        <SRDF>{exotica_examples}/resources/robots/lwr_simplified.srdf</SRDF>
+      </Scene>
+    </PlanningScene>
+    
+    <Maps>
+      <EffFrame Name="Position">
+        <EndEffector>
+            <Frame Link="lwr_arm_6_link" LinkOffset="0 0 0 0.7071067811865476 -4.3297802811774664e-17  0.7071067811865475 4.3297802811774664e-17"/>
+        </EndEffector>
+      </EffFrame>
+	<EffBoxConstraint Name="EffBox" XLim="-3 3" YLim="-0.1 3" ZLim="0 0.65">
+	  <EndEffector>
+	    <Frame Link="lwr_arm_6_link"/>
+	  </EndEffector>
+	</EffBoxConstraint>
+    </Maps>
+
+    <Cost>
+      <Task Task="Position"/>
+    </Cost>
+
+    <Equality>
+    </Equality>
+    
+    <Inequality>
+      <Task Task="EffBox"/>
+    </Inequality>
+
+    <UseBounds>0</UseBounds>
```

Comment:
Why?

Reply:
When `UseBounds=0` see [here](https://youtu.be/HRJ9bSm9OfE) and when `UseBounds=1` see [here](https://youtu.be/FKq7VJ8y9ZQ). Both use the same starting state. 

I see this sort of issue with other examples, my suspicion is that the figure of eight example is defined in a particular position that pushes the kuka towards its joint limits resulting in this strange configuration that it is unable to get out of.